docs/modules/ROOT/partials/install/bootstrap-nodes.adoc

```diff
@@ -0,0 +1,202 @@
+
+. Deploy bootstrap node
++
+[source,bash]
+----
+cat > override.tf <<EOF
+module "cluster" {
+  bootstrap_count = 1
+  master_count    = 0
+  infra_count     = 0
+  storage_count   = 0
```

Comment:
In this partial, the variable `storage_count` is relevant for exoscale, but not for cloudscale. However, this variable doesn't hurt for cloudscale either. Shall I do If/else's or leave it?

Reply:
I'd ensure that `storage_count=0` is only present for Exoscale to minimize the potential for confusion